qemu/tests/cfg/qmp_command.cfg

```diff
@@ -186,3 +186,7 @@
         - qmp_query-target:
             qmp_cmd = "query-target"
             cmd_result_check = equal
+        - qmp_query-machines:
+            qmp_cmd = "query-machines"
+            cmd_result_check = "contain"
+            cmd_return_value = "['name', 'cpu-max']"
```

Comment:
Please distinguish the expect value for different version.

Reply:
After checked  the output for different version, cover both RHEL7(qemu-kvm,qemu-kvm-rhev) and RHEL8(slow and fast).
The current checking is ok for me. some other key, such as `hotpluggable-cpus`, `cpu_max` can be covered by other cases. No risk as my understanding!
@jingzhao84 Any other comments for this issue ?

Reply:
@yiqianwei @yanan-fu 
This qmp command should focus on machine type check. so i think it should contain "name": "pc-q35-rhel8.2.0" check.
such as for x86_64: contain i440fx, q35, for ppc should contian pseries
How do you thinks?

Thanks
Jing

Reply:
@jingzhao84 @yanan-fu 
I have modified this patch, Could you help to review this patch again ? thanks

Reply:
With this fix, it only check the machine type the vm boot with is in the qmp output.
In fact, it will always be true. otherwise vm can not boot.
@jingzhao84  what's your opinion for this ?

Reply:
@yanan-fu  I think it is ok.